app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+
+#### Salary requirement for a skilled worker visa 
+
+The salary threshold you must meet depends on the region you work in as a salaried trainee.
+
+#### Full-time training
+
+Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a skilled worker visa – this varies by region, as follows:
+
+* inner London: £23,099
+* outer London: £21,832
+* London fringe: £19,613
+* rest of England: £18,419
+
+Check the region you will be working in with your teacher training provider.
+
+#### Part-time training
+
+You may be able to train part-time, but your salary must meet the threshold for a skilled worker visa.
+
+There are 2 requirement:
+
+Your part-time salary must be at least £20,480.
+
+The full-time equivalent of your salary must meet the threshold set for the skilled worker visa in your region, as follows: 
+
+* inner London: £23,099
+* outer London: £21,832
+* London fringe: £19,613
+* rest of England: £18,419
+
+Check the region you will be working in with your teacher training provider.
+
+<p class="call-to-action__action">
+  <a href="https://www.gov.uk/skilled-worker-visa">Apply for a skilled worker visa<span></span></a>
+</p>
+
+### Postgraduate teaching apprenticeships
+
+To apply for a postgraduate teaching apprenticeship you will need:
+
+* settled or pre-settled status under the EU Settlement Scheme
+* to have been living in the UK or the EEA for 3 or more years before the start of your apprenticeship
+
+If you do not have settled or pre-settled status under the EU Settlement Scheme, you will need:
+
+* a visa or immigration status allowing you to work in the UK
+* to have been living in the UK for 3 or more years before the start of your apprenticeship
+
+## Other types of visa you can use to work and study in England
+
+### Graduate visa
+
+If you are already an international student in the UK, you can [apply for a graduate visa on successful completion of your degree](https://www.gov.uk/graduate-visa).
+
+To apply, you will need to have been sponsored by a Home Office licensed student sponsor over the course of your studies.
+
+The graduate visa will allow you to study, work, or look for work in the UK for up to 2 years after completing your studies (3 years for PhD students) without a sponsor. 
+
+This includes training as a teacher on a salaried or fee-paying initial teacher training course, if the course does not meet the requirements for a student visa. If the course can sponsor a student visa, you must apply for this type of visa.
+
+You will be able to switch from a graduate visa to another visa (such as the skilled worker visa) without having to leave the UK, if you meet the requirements for the new visa route.
+
+### High potential individual visa
+
+[To be eligible for a high potential individual visa](https://www.gov.uk/high-potential-individual-visa), you must:
```

Comment:
I would move the link to the end of the section.

Reply:
OK